misc/inplace-testing/make-and-prep.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -ex
```

Comment:
Any reason why `misc/` and not `tests/` or `edb/testbase/`?

Reply:
Because it isn't part of the real test suite, I guess. I can move it though.